admin/services/nca-tools-user.service.js

```diff
@@ -49,12 +49,6 @@ const service = {
     userRecord.mtcRole = roleService.mapNcaRoleToMtcRole(ncaUser.UserType, school)
     return userRecord
   },
-  recordLogonAttempt: async (logonData) => {
```

Comment:
is this supposed to be removed?

Reply:
Please see my response to Guy's question about `admin/spec/service/nca-tools-user.service.spec.js` above.